Core/Src/main.c

```diff
@@ -767,6 +769,53 @@ static void onboot_commands(void) {
 #endif
 #endif
 }
+
+/*
+ * @brief Store information from existing files from NAND to
+ * 		  RAM (buffer)
+ *
+ * 		  The purpose of transferring file info (file_id, file_name,
+ * 		  file_size) from NAND to RAM during on-boot is to set up a
+ * 		  quick lookup table for image transfer APIs to quickly
+ * 		  extract file information, instead of accessing NAND structures
+ * 		  directly.
+ */
+static int store_file_infos_in_buffer() {

```

Comment:
Shouldn't this be in nandfs? I don't like how it uses an extern

Reply:
Yeah, you correct. I have moved this the command_handler where the buffer is created